apply/apply_test.go

```diff
@@ -224,48 +225,75 @@ func TestApplySmokeTest(t *testing.T) {
 	r.NoError(err)
 	// defer os.RemoveAll(d)
 
-	json := `
-{
-  "defaults": {
-    "aws_region_provider": "reg",
-    "aws_region_backend": "reg",
-    "aws_profile_provider": "prof",
-    "aws_profile_backend": "prof",
-    "aws_provider_version": "0.12.0",
-    "account_id": 789,
-    "infra_s3_bucket": "buck",
-    "project": "proj",
-    "terraform_version": "0.100.0",
-    "owner": "foo@example.com"
-  },
-  "travis_ci": {
-	"enabled": true,
-	"aws_iam_role_name": "travis",
-        "id_account_name": "id",
-        "test_buckets": 7
-  },
-  "accounts": {
-    "foo": {
-      "account_id": 123
-    },
-    "bar": {
-      "account_id": 456
-    }
-  },
-  "modules": {
-    "my_module": {}
-  },
-  "envs": {
-    "staging":{
-        "components": {
-            "comp1": {},
-            "comp2": {}
-        }
-    },
-    "prod": {}
-  }
-}
+	yaml := `
+defaults:
+  owner: foo
+  project: bar
+  terraform_version: 0
+  backend:
+    bucket: baz
+    region: qux
+    profile: quux
+version: 2
 `
+	c, e := v2.ReadConfig([]byte(yaml))
+	r.NoError(e)
+
+	w, e := c.Validate()
+	r.NoError(e)
+	r.Len(w, 0)
+
+	e = Apply(fs, c, templates.Templates, false)
+	r.NoError(e)
+}
+
+func TestApplySmokeTestJSON(t *testing.T) {
+	r := require.New(t)
+	fs, _, err := util.TestFs()
+	r.NoError(err)
+	// defer os.RemoveAll(d)
```

Comment:
where did this come from?

Reply:
This was in the original codebase, but I can remove it

Reply:
thanks, I think keeping it there is confusing since it doesn't look like we even have `d` variable in scope